control/controlclient/netmap.go

```diff
@@ -26,6 +26,7 @@ type NetworkMap struct {
 	NodeKey       tailcfg.NodeKey
 	PrivateKey    wgcfg.PrivateKey
 	Expiry        time.Time
+	Name          string
```

Comment:
comments on this field please. Name is pretty ambiguous.

(admittedly many fields here don't have docs, but Boy Scout Rule: https://deviq.com/boy-scout-rule/)